pykeybasebot/bot.py

```diff
@@ -131,18 +131,18 @@ def kvstore(self):
 
     async def _is_initialized(self):
         if not self._initialized:
-            res = await self.submit("status --json")
+            res = await self.submit("whoami --json")
```

Comment:
doesn't look like `whoami --json` works. maybe we drop the json and read the result right into username?

Reply:
need to update your client

Reply:
https://github.com/keybase/client/pull/22699

Reply:
:+1: then this needs to come with a version bump, and we can't release it until the keybase client has actually been released.